frontend/src/container/GroupedNotificationItem.jsx

```diff
@@ -17,124 +17,148 @@ import {
-        ? content.parentId
-        : content.id
-      )
+    const i18nOpts = {
+      user: `<span title='${escapedUser}'>${escapedUser}</span>`,
+      author: `<span title='${escapedAuthorList}'>${escapedAuthorList}</span>`,
+      content: `<span title='${escapedContentLabel}' class='contentTitle__highlight'>${escapedContentLabel}</span>`,
+      numberOfContribution: notification.group.length,
+      numberOfWorkspaces: numberOfWorkspaces,
+      interpolation: { escapeValue: false }
     }
 
-    return {
-      text: numberOfWorkspaces > 1
-        ? t('{{author}} made {{numberOfContribution}} contributions in {{numberOfWorkspaces}} spaces', i18nOpts)
-        : numberOfContents === 1
-          ? t('{{author}} made {{numberOfContribution}} contributions on {{content}}', i18nOpts)
-          : t('{{author}} made {{numberOfContribution}} contributions', i18nOpts),
-      url: contentUrl
+    if (numberOfContributionTypes > 1) {
+      let contentUrl
+      if (numberOfContents === 1) {
+        const content = notification.group.find(notification => notification.content).content
+        contentUrl = PAGE.CONTENT(content.type === TLM_SUB.COMMENT
+          ? content.parentId
+          : content.id
+        )
+      }
+
+      return {
+        text: numberOfWorkspaces > 1
+          ? props.t('{{author}} made {{numberOfContribution}} contributions in {{numberOfWorkspaces}} spaces', i18nOpts)
+          : numberOfContents === 1
+            ? props.t('{{author}} made {{numberOfContribution}} contributions on {{content}}', i18nOpts)
+            : props.t('{{author}} made {{numberOfContribution}} contributions', i18nOpts),
+        url: contentUrl
+      }
+    } else {
+      const notificationDetails = props.getNotificationDetails({
+        ...notification.group[0],
+        author: { publicName: escapedAuthorList },
+        content: { ...notification.group[0].content, label: escapedContentLabel, parentLabel: escapedContentLabel },
+        numberOfWorkspaces: numberOfWorkspaces,
+        numberOfContents: numberOfContents,
+        type: notification.group[0].type,
+        user: { ...notification.group[0].user, publicName: escapedUser }
+      })
+
+      if (numberOfContents !== 1) delete notificationDetails.url
+      return notificationDetails
     }
-  } else {
-    const notificationDetails = getNotificationDetails({
-      ...notification.group[0],
-      author: { publicName: escapedAuthorList },
-      content: { ...notification.group[0].content, label: escapedContentLabel, parentLabel: escapedContentLabel },
-      numberOfWorkspaces: numberOfWorkspaces,
-      numberOfContents: numberOfContents,
-      type: notification.group[0].type,
-      user: { ...notification.group[0].user, publicName: escapedUser }
-    })
+  }
 
-    if (numberOfContents !== 1) delete notificationDetails.url
-    return notificationDetails
+  // FIXME - MP - 14-03-2022 - Function removed so I can safely push
```

Comment:
Please, make an issue about this and link with this comment

Reply:
Updated with #5526